test/unit/post_out_of_sync_deploys_service_test.rb

```diff
@@ -0,0 +1,87 @@
+require "test_helper"
+
+class PostOutOfSyncDeploysServiceTest < ActiveSupport::TestCase
+  describe "#call" do
+    should "call FindOutOfSyncDeploysService to get a list of out-of-sync deploys" do
+      teams_out_of_sync_deploys = {
+        "#govuk-publishing-platform" =>
+          [{ name: "Asset manager",
+             repo: "alphagov/asset-manager",
+             status: :undeployed_changes_in_integration,
+             dependencies_team: "#govuk-publishing-platform" }],
+      }
+
+      find_service_mock = MiniTest::Mock.new
+      find_service_mock.expect(:call, teams_out_of_sync_deploys)
+
+      slack_poster_mock = MiniTest::Mock.new
+      expected_args = [
+        "Hello :paw_prints:, this is your regular badgering to deploy!\n" \
+        "\n" \
+        "- <https://github.com/alphagov/asset-manager/actions/workflows/deploy.yml|Asset manager> – Undeployed changes in integration",
+        "#govuk-publishing-platform",
+        { "icon_emoji" => ":badger:" },
+      ]
+      slack_poster_mock.expect(:call, nil, expected_args)
+
+      FindOutOfSyncDeploysService.stub(:call, find_service_mock) do
+        SlackPosterWorker.stub(:perform_async, slack_poster_mock) do
+          PostOutOfSyncDeploysService.call
+          assert_mock find_service_mock
+        end
+      end
+    end
+
+    should "call SlackPosterWorker to post messages with correct payload for each team with out-of-sync deploys" do
```

Comment:
You reference `stringify_keys` in the commit message, but I don't see it anywhere in the code?

Reply:
Good spot, I changed this method but forgot to update the commit message